### ISSUE TITLE
packer validate fail with "Unknown region: ap-northeast-2"

### ISSUE THREAD

**Issue by giseongeom** [mitchellh/packer#3058]
This week, AWS new region [ap-northeast-2] (https://aws.amazon.com/ko/blogs/aws/now-open-aws-asia-pacific-seoul-region/) is announced. It seems like packer support for 'ap-northeast-2' not ready.

* Environment
 * Host OS:  Cygwin64 on Windows 10 Build 1511
 * builder: amazon-ebs
 * Packer version: 0.8.6
 * Detailed log: See belows or  [my gist] (https://gist.github.com/giseongeom/10ca15da63d3fa204acc)
```
[x86_64-unknown-cygwin] giseong.eom@GISEONG-PC ~/home/Documents/packer-getting-started
$ PACKER_LOG=1 packer validate create-1st-custom-ami.json
2016/01/12 11:14:43 [INFO] Packer version: 0.8.6  f8f7b7a34c1be06058f5aca23a51247db12cdbc5+CHANGES
2016/01/12 11:14:43 Packer Target OS/Arch: windows amd64
2016/01/12 11:14:43 Built with Go Version: go1.5
2016/01/12 11:14:43 [DEBUG] Discovered plugin: amazon-chroot = C:\App\Vagrant\bin\packer-builder-amazon-chroot.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: amazon-ebs = C:\App\Vagrant\bin\packer-builder-amazon-ebs.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: amazon-instance = C:\App\Vagrant\bin\packer-builder-amazon-instance.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: digitalocean = C:\App\Vagrant\bin\packer-builder-digitalocean.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: docker = C:\App\Vagrant\bin\packer-builder-docker.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: file = C:\App\Vagrant\bin\packer-builder-file.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: googlecompute = C:\App\Vagrant\bin\packer-builder-googlecompute.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: null = C:\App\Vagrant\bin\packer-builder-null.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: openstack = C:\App\Vagrant\bin\packer-builder-openstack.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: parallels-iso = C:\App\Vagrant\bin\packer-builder-parallels-iso.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: parallels-pvm = C:\App\Vagrant\bin\packer-builder-parallels-pvm.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: qemu = C:\App\Vagrant\bin\packer-builder-qemu.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: virtualbox-iso = C:\App\Vagrant\bin\packer-builder-virtualbox-iso.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: virtualbox-ovf = C:\App\Vagrant\bin\packer-builder-virtualbox-ovf.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: vmware-iso = C:\App\Vagrant\bin\packer-builder-vmware-iso.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: vmware-vmx = C:\App\Vagrant\bin\packer-builder-vmware-vmx.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: artifice = C:\App\Vagrant\bin\packer-post-processor-artifice.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: atlas = C:\App\Vagrant\bin\packer-post-processor-atlas.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: compress = C:\App\Vagrant\bin\packer-post-processor-compress.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: docker-import = C:\App\Vagrant\bin\packer-post-processor-docker-import.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: docker-push = C:\App\Vagrant\bin\packer-post-processor-docker-push.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: docker-save = C:\App\Vagrant\bin\packer-post-processor-docker-save.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: docker-tag = C:\App\Vagrant\bin\packer-post-processor-docker-tag.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: vagrant-cloud = C:\App\Vagrant\bin\packer-post-processor-vagrant-cloud.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: vagrant = C:\App\Vagrant\bin\packer-post-processor-vagrant.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: vsphere = C:\App\Vagrant\bin\packer-post-processor-vsphere.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: ansible-local = C:\App\Vagrant\bin\packer-provisioner-ansible-local.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: chef-client = C:\App\Vagrant\bin\packer-provisioner-chef-client.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: chef-solo = C:\App\Vagrant\bin\packer-provisioner-chef-solo.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: file = C:\App\Vagrant\bin\packer-provisioner-file.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: powershell = C:\App\Vagrant\bin\packer-provisioner-powershell.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: puppet-masterless = C:\App\Vagrant\bin\packer-provisioner-puppet-masterless.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: puppet-server = C:\App\Vagrant\bin\packer-provisioner-puppet-server.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: salt-masterless = C:\App\Vagrant\bin\packer-provisioner-salt-masterless.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: shell-local = C:\App\Vagrant\bin\packer-provisioner-shell-local.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: shell = C:\App\Vagrant\bin\packer-provisioner-shell.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: windows-restart = C:\App\Vagrant\bin\packer-provisioner-windows-restart.exe
2016/01/12 11:14:43 [DEBUG] Discovered plugin: windows-shell = C:\App\Vagrant\bin\packer-provisioner-windows-shell.exe
2016/01/12 11:14:43 Attempting to open config file: C:\Users\giseong.eom\AppData\Roaming\packer.config
2016/01/12 11:14:43 [WARN] Config file doesn't exist: C:\Users\giseong.eom\AppData\Roaming\packer.config
2016/01/12 11:14:43 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[amazon-chroot:C:\App\Vagrant\bin\packer-builder-amazon-chroot.exe googlecompute:C:\App\Vagrant\bin\packer-builder-googlecompute.exe openstack:C:\App\Vagrant\bin\packer-builder-openstack.exe virtualbox-iso:C:\App\Vagrant\bin\packer-builder-virtualbox-iso.exe digitalocean:C:\App\Vagrant\bin\packer-builder-digitalocean.exe docker:C:\App\Vagrant\bin\packer-builder-docker.exe file:C:\App\Vagrant\bin\packer-builder-file.exe parallels-iso:C:\App\Vagrant\bin\packer-builder-parallels-iso.exe qemu:C:\App\Vagrant\bin\packer-builder-qemu.exe virtualbox-ovf:C:\App\Vagrant\bin\packer-builder-virtualbox-ovf.exe vmware-iso:C:\App\Vagrant\bin\packer-builder-vmware-iso.exe vmware-vmx:C:\App\Vagrant\bin\packer-builder-vmware-vmx.exe null:C:\App\Vagrant\bin\packer-builder-null.exe parallels-pvm:C:\App\Vagrant\bin\packer-builder-parallels-pvm.exe amazon-ebs:C:\App\Vagrant\bin\packer-builder-amazon-ebs.exe amazon-instance:C:\App\Vagrant\bin\packer-builder-amazon-instance.exe] PostProcessors:map[atlas:C:\App\Vagrant\bin\packer-post-processor-atlas.exe docker-import:C:\App\Vagrant\bin\packer-post-processor-docker-import.exe docker-tag:C:\App\Vagrant\bin\packer-post-processor-docker-tag.exe vsphere:C:\App\Vagrant\bin\packer-post-processor-vsphere.exe artifice:C:\App\Vagrant\bin\packer-post-processor-artifice.exe compress:C:\App\Vagrant\bin\packer-post-processor-compress.exe docker-push:C:\App\Vagrant\bin\packer-post-processor-docker-push.exe docker-save:C:\App\Vagrant\bin\packer-post-processor-docker-save.exe vagrant-cloud:C:\App\Vagrant\bin\packer-post-processor-vagrant-cloud.exe vagrant:C:\App\Vagrant\bin\packer-post-processor-vagrant.exe] Provisioners:map[file:C:\App\Vagrant\bin\packer-provisioner-file.exe puppet-masterless:C:\App\Vagrant\bin\packer-provisioner-puppet-masterless.exe puppet-server:C:\App\Vagrant\bin\packer-provisioner-puppet-server.exe salt-masterless:C:\App\Vagrant\bin\packer-provisioner-salt-masterless.exe shell-local:C:\App\Vagrant\bin\packer-provisioner-shell-local.exe ansible-local:C:\App\Vagrant\bin\packer-provisioner-ansible-local.exe chef-client:C:\App\Vagrant\bin\packer-provisioner-chef-client.exe chef-solo:C:\App\Vagrant\bin\packer-provisioner-chef-solo.exe powershell:C:\App\Vagrant\bin\packer-provisioner-powershell.exe shell:C:\App\Vagrant\bin\packer-provisioner-shell.exe windows-restart:C:\App\Vagrant\bin\packer-provisioner-windows-restart.exe windows-shell:C:\App\Vagrant\bin\packer-provisioner-windows-shell.exe]}
2016/01/12 11:14:43 Setting cache directory: C:\Users\giseong.eom\Documents\packer-getting-started\packer_cache
2016/01/12 11:14:43 Loading builder: amazon-ebs
2016/01/12 11:14:43 Creating plugin client for path: C:\App\Vagrant\bin\packer-builder-amazon-ebs.exe
2016/01/12 11:14:43 Starting plugin: C:\App\Vagrant\bin\packer-builder-amazon-ebs.exe []string{"C:\\App\\Vagrant\\bin\\packer-builder-amazon-ebs.exe"}
2016/01/12 11:14:43 Waiting for RPC address for: C:\App\Vagrant\bin\packer-builder-amazon-ebs.exe
2016/01/12 11:14:43 packer-builder-amazon-ebs.exe: 2016/01/12 11:14:43 Plugin minimum port: 10000
2016/01/12 11:14:43 packer-builder-amazon-ebs.exe: 2016/01/12 11:14:43 Plugin maximum port: 25000
2016/01/12 11:14:43 packer-builder-amazon-ebs.exe: 2016/01/12 11:14:43 Plugin address: tcp 127.0.0.1:10000
2016/01/12 11:14:43 packer-builder-amazon-ebs.exe: 2016/01/12 11:14:43 Waiting for connection...
2016/01/12 11:14:43 packer-builder-amazon-ebs.exe: 2016/01/12 11:14:43 Serving a plugin connection...
2016/01/12 11:14:43 Loading provisioner: shell
2016/01/12 11:14:43 Creating plugin client for path: C:\App\Vagrant\bin\packer-provisioner-shell.exe
2016/01/12 11:14:43 Starting plugin: C:\App\Vagrant\bin\packer-provisioner-shell.exe []string{"C:\\App\\Vagrant\\bin\\packer-provisioner-shell.exe"}
2016/01/12 11:14:43 Waiting for RPC address for: C:\App\Vagrant\bin\packer-provisioner-shell.exe
2016/01/12 11:14:43 packer-provisioner-shell.exe: 2016/01/12 11:14:43 Plugin minimum port: 10000
2016/01/12 11:14:43 packer-provisioner-shell.exe: 2016/01/12 11:14:43 Plugin maximum port: 25000
2016/01/12 11:14:43 packer-provisioner-shell.exe: 2016/01/12 11:14:43 Plugin address: tcp 127.0.0.1:10000
2016/01/12 11:14:43 packer-provisioner-shell.exe: 2016/01/12 11:14:43 Waiting for connection...
2016/01/12 11:14:43 packer-provisioner-shell.exe: 2016/01/12 11:14:43 Serving a plugin connection...
2016/01/12 11:14:43 Preparing build: amazon-ebs
2016/01/12 11:14:43 Build 'amazon-ebs' prepare failure: 1 error(s) occurred:

* Unknown region: ap-northeast-2
2016/01/12 11:14:43 ui error: Template validation failed. Errors are shown below.
2016/01/12 11:14:43 ui error: Errors validating build 'amazon-ebs'. 1 error(s) occurred:

* Unknown region: ap-northeast-2
2016/01/12 11:14:43 waiting for all plugin processes to complete...
Template validation failed. Errors are shown below.

Errors validating build 'amazon-ebs'. 1 error(s) occurred:

* Unknown region: ap-northeast-2
2016/01/12 11:14:43 C:\App\Vagrant\bin\packer-builder-amazon-ebs.exe: plugin process exited
2016/01/12 11:14:43 C:\App\Vagrant\bin\packer-provisioner-shell.exe: plugin process exited
[x86_64-unknown-cygwin] giseong.eom@GISEONG-PC ~/home/Documents/packer-getting-started
```

**Comment by jen20** [comment]
Hi @giseongeom! I've fixed this in #3059, but it will need a member of the Packer team to review it prior to merging. Thanks for opening the issue!

**Comment by giseongeom** [comment]
@jen20 Thanks, I'll close this duplicated issue.
